bedrock/base/views.py

```diff
@@ -83,13 +90,33 @@ def get_extra_server_info():
         pass
     else:
         db_info["last_update"] = timeago.format(datetime.fromtimestamp(db_info["updated"]))
+        db_info["last_updated_timestamp"] = datetime.fromtimestamp(db_info["updated"])
```

Comment:
style nit: could refactor to get the timestamp first, then use it for both values:

```python
last_updated_timestamp = datetime.fromtimestamp(db_info["updated"])
db_info["last_updated_timestamp"] = last_updated_timestamp
db_info["last_update"] = timeago.format(last_updated_timestamp)
```